### PR TITLE
form_builder_phone_field update

### PR DESCRIPTION
**Pull request by deandreamatias**
## Connection with issue(s)

Connected to #855

## Testing and Review Notes
 
- Test with Flutter 2.8.1 and 2.10.5

Next steps:
- Publish to pub.dev with version 1.0.0

## Screenshots or Videos

image
image

## To Do

- [ ] double check the original issue to confirm it is fully satisfied
- [X] add testing notes and screenshots in PR description to help guide reviewers
- [ ] request the "UX" team perform a design review (if/when applicable)

### CODE REVIEW

File: packages/form_builder_phone_field/.fvm/fvm_config.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "flutterSdkVersion": "2.10.5",
+  "flavors": {}
+}
```
Comment (WilliamCunhaCardoso) [inline packages/form_builder_phone_field/.fvm/fvm_config.json:4]: Should we submit `fvm` environment subject? I am not sure about it.

Comment (deandreamatias) [reply packages/form_builder_phone_field/.fvm/fvm_config.json:4]: I don't know too :(
I feel it is difficult to know which versions support each packages or which ones they are built with. Maybe this isn't better solution for that

**Comment by danvick** [comment]
@deandreamatias are we done here?

**Comment by deandreamatias** [comment]
Yes! Nice changes

After this, I will do another pull request to be compatible with last version of flutter_form_builder and Flutter 3.0. But this is the future...

This PR will be merge

**Pull request merged**